backend/tests/unit/conftest.py

```diff
@@ -1,9 +1,17 @@
 import pytest
 
-from ....common.annotations import YieldFixture
+from ..common.annotations import YieldFixture
 from . import fakers
 
 
+@pytest.fixture
+def test_extract_unit_of_work() -> YieldFixture[fakers.TestExtractUnitOfWork]:
+    test_unit_of_work = fakers.TestExtractUnitOfWork()
+    test_unit_of_work.file = fakers.TestFileDomainRepository()
```

Comment:
Well, you should never explicitly define repositories like this.

TestExtractUnitOfWork must have them.
Something like this:

```python
class TestEventsUnitOfWork(domain_ports.AbstractEventsUnitOfWork):
    def __init__(self):
        self.events = TestEventsDomainRepository()
        self.reservations = TestReservationsDomainRepository()

    def __enter__(self):
        return self

    def __exit__(self, exc_type, exc_val, exc_tb):
        pass

    def commit(self):
        pass

    def rollback(self):
        pass

```